unittest/test_ZstdCompression.cpp

```diff
@@ -90,8 +90,8 @@ TEST_CASE("Large compressible Compression::Type::zstd roundtrip")
 TEST_CASE("Large uncompressible Compression::Type::zstd roundtrip")
 {
   char data[100000];
-  for (size_t i = 0; i < sizeof(data); i++) {
-    data[i] = rand() % 256;
+  for (char& i : data) {
+    i = rand() % 256;
```

Comment:
No thanks. :slightly_smiling_face: But if you must, name the variable `c` or `ch`.